docs/getting-started.md

```diff
@@ -43,44 +43,93 @@ select target VMs, and import VMs into the K8s cluster as `VirtualMachine` CRs.
 ### CloudProviderAccount
 
 To import cloud VMs, user needs to configure a `CloudProviderAccount` CR, with
-the cloud account credentials.
+a K8s secret containing base64 encoded cloud account credentials. The secret
+should be created in `nephe-system` namespace so nephe controller can access it.
 
-* Sample `CloudProviderAccount` for AWS:
+#### Sample `Secret` for AWS:
+
+To get the base64 encoded json string for credential, run:
+
+```bash
+echo '{"accessKeyId": "YOUR_AWS_ACCESS_KEY_ID", "accessKeySecret": "YOUR_AWS_ACCESS_KEY_SECRET", "roleArn": "YOUR_AWS_IAM_ROLE_ARN", "externalId": "IAM_ROLE_EXTERNAL_ID"}' | openssl base64
```

Comment:
Don't really agree here. All our code blocks, links, terminal outputs retain its original length, even if they exceed char limit. Also for code block render, it will have horizontal scroll on this particular block if its too long.